Week3/Contributor.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+class Contributor {
+  constructor(data) {
+    this.data = data;
+  }
+
+  render(parent) {
+    parent.innerHTML = '';
+
+    try {
+      for (let prop in this.data) {
+        const liContribInfo = Util.createHTMLElement('li', parent);
+        let [...contribDetails] = liContribInfo.children;
+        contribDetails[0] = Util.createHTMLElement('a', liContribInfo);
+        const contributorsAvatar = Util.createHTMLElement('img', contribDetails[0]);
+        contribDetails[1] = Util.createHTMLElement('li', liContribInfo, 'Username: '.bold() + this.data[prop].login);
```

Comment:
Don't use this `.bold()` string method. It is deprecated: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/bold

Reply:
I personally did not want to use those too, but unfortunately I could not find any useful information on how to make the first word bold, CSS does not have such property and the majority of Javascript solutions are either not clear enough or are presented using jQuery. Do you maybe have any suggestions?

Reply:
You can either embed the word to made bold with  the standard element `<strong>`  which is by default styled to be bold or you can use a `<span>` with a class that you style yourself and set the `font-weight: bold;`. As your `createHTMLElement()` function uses the third parameter to set  the `innerHTML` of the element, you could simple do something like this:

```
contribDetails[1] = Util.createHTMLElement('li', liContribInfo, `<strong>Username:</strong> ${contributor.login}`);
```

I haven't tried it but I think it should work.

Reply:
Yes this does work :+1:  interesting , before using bold() I did this  '<strong> Username: </strong>' + contributor.login, and  it did not work. I guess it has something to do with the function I used. Thanks for this solution!